### PR TITLE
add columns to app grid

### PR DESCRIPTION
**Pull request by shmuliko**
* takd id to active and closed
* create date to active
* exec date to closed

### CODE REVIEW

File: protected/templates/admin/en/apps/ajax/app_tbody.php [hunk 1]
```diff
@@ -1,5 +1,19 @@
 <?php foreach ($tasks as $task): ?>
     <tr>
+        <td>
+            <?php if ($type == 2){
+                echo $task['task_id'];
+            } else {
+                echo $task['id'];
+            }?>
```
Comment (mosheka) [inline protected/templates/admin/en/apps/ajax/app_tbody.php]: why not a single line:
echo fi ($type == 2) ? $task['task_id'] : $task['id'];

File: protected/templates/admin/en/apps/ajax/app_tbody.php [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+        </td>
+        <td>
+            <?php if ($type == 2){
+                echo $task['send_date'];
+            } else {
+                echo $task['create_date'];
+            }?>
+        </td>
         <td>
```
Comment (mosheka) [inline protected/templates/admin/en/apps/ajax/app_tbody.php]: same

File: protected/templates/admin/en/apps/ajax/app_tbody.php (hunk 1 above)
Comment (shmuliko) [reply protected/templates/admin/en/apps/ajax/app_tbody.php]: i fix this

File: protected/templates/admin/en/apps/ajax/app_tbody.php (hunk 2 above)
Comment (shmuliko) [reply protected/templates/admin/en/apps/ajax/app_tbody.php]: some fix

**Pull request merged**